src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -685,30 +749,34 @@ private class ACCT_IndividualAccounts_TEST {
         Contact con = contacts[0];
         insert con;
 
-        Account assertAccount = [SELECT Id, RecordType.Name, Name FROM Account limit 1];
-        system.assertEquals('Household Account', assertAccount.RecordType.Name);
-        system.assertEquals(con.FirstName + ' ' + con.LastName + ' Household', assertAccount.Name);
+        Account assertAccount = [SELECT Id, RecordType.Name, Name
+                                 FROM Account LIMIT 1];
+        System.assertEquals('Household Account', assertAccount.RecordType.Name);
+        System.assertEquals(con.FirstName + ' ' + con.LastName + ' Household', assertAccount.Name);
 
         Contact con2 = contacts[1];
         con2.AccountId = assertAccount.Id;
         con2.FirstName = 'Test2';
         con2.LastName = con.LastName + '2';
         insert con2;
 
-        assertAccount = [SELECT Id, RecordType.Name, Name FROM Account];
+        assertAccount = [SELECT Id, RecordType.Name, Name
+                         FROM Account];
 
-        system.assertEquals('Household Account', assertAccount.RecordType.Name);
-        system.assertEquals(con.FirstName + ' ' + con.LastName + ' and ' + con2.FirstName + ' ' + con2.LastName + ' Household', assertAccount.Name);
+        System.assertEquals('Household Account', assertAccount.RecordType.Name);
+        System.assertEquals(con.FirstName + ' ' + con.LastName + ' and ' + con2.FirstName + ' ' + con2.LastName + ' Household', assertAccount.Name);
 
         con2.AccountId = null;
         Test.startTest();
         update con2;
         Test.stopTest();
 
         //After remove the Account lookup for con2, system will create a new Account for con2. So, we need to query the Account of con in order to verify the Account Name.
-        assertAccount = [SELECT Id, RecordType.Name, Name FROM Account where Id =: assertAccount.Id];
-        system.assertEquals('Household Account', assertAccount.RecordType.Name);
-        system.assertEquals(con.FirstName + ' ' + con.LastName + ' Household', assertAccount.Name);
+        assertAccount = [SELECT Id, RecordType.Name, Name
+                         FROM Account
+                         WHERE Id =: assertAccount.Id];
```

Comment:
nit: just pointing out here that in some places we have "=: something" and in others we have "= :something" -- nothing I'd hold up a merge for.

Reply:
@lifewithryan , no prob! I'll clean it up.